src/components/PostDetails.tsx

```diff
@@ -1,106 +1,76 @@
-import React from 'react';
-import { Loader } from './Loader';
-import { NewCommentForm } from './NewCommentForm';
+import React, { useEffect, useState } from 'react';
+
+import { Loader, CommentItem, NewCommentForm } from './index';
+import { Post } from '../types';
+import { useComments } from '../hooks/useComments';
+
+type Props = {
+  selectedPost: Post;
+};
+
+export const PostDetails: React.FC<Props> = ({ selectedPost }) => {
+  const { id, title, body } = selectedPost;
+
+  const [isFormOpened, setisFormOpened] = useState(false);
```

Comment:
The state variable `setisFormOpened` should be renamed to `setIsFormOpened` to maintain consistent camelCase naming conventions.